tools/actions/commit-queue.sh

```diff
@@ -76,9 +79,10 @@ for pr in "$@"; do
     git node land --abort --yes
   else
     rm output
-    git push origin master
+    git push $UPSTREAM $DEFAULT_BRANCH
 
-    gitHubCurl "$(commentsUrl "$pr")" POST --data '{"body": "Landed in '"$(git rev-parse HEAD)"'"}'
+    commits="$(git merge-base $UPSTREAM/$DEFAULT_BRANCH HEAD)...$(git rev-parse HEAD)"
```

Comment:
Isn't `$UPSTREAM/$DEFAULT_BRANCH` already at the same commit as `HEAD` at this point?

Reply:
Oh, that's right, it should be done before the push :+1: